src/components/PDFViewer/PDFViewer.scss

```diff
@@ -2,3 +2,20 @@
   padding-top: 129.41%; /* about 11:8.5 standard page size */
 }
 
+@media screen and (min-device-aspect-ratio: 4/3) {
+  .responsive-pdf::before{
+    padding-top: 75%;
+  }
+}
+
+@media screen and (min-device-aspect-ratio: 16/10) {
+  .responsive-pdf::before{
+    padding-top: 62%;
+  }
+}
+
+@media screen and (min-device-aspect-ratio: 16/9) {
+  .responsive-pdf::before{
+    padding-top: 56%;
```

Comment:
change this to 85% as it makes it taller on desktop screens and fill most of the screen which is what we want